src/formula/callable_objects.cpp

```diff
@@ -849,4 +860,87 @@ void safe_call_result::get_inputs(formula_input_vector& inputs) const
 	}
 }
 
+void gamestate_callable::get_inputs(formula_input_vector &inputs) const
+{
+	add_input(inputs, "turn");
+	add_input(inputs, "time_of_day");
+	add_input(inputs, "current_side");
```

Comment:
I'm wondering about what I should call the turn and side numbers here. In Lua they are `wesnoth.current.turn` and `wesnoth.current.side`, while in WML they are `$turn_number` and `$side_number`. That suggests three possibilities for WFL:

* `turn` and `side` (I can't remember if there was a reason I didn't use `side`…)
* `current_turn` and `current_side` (like Lua but that dot becomes an underscore)
* `turn_number` and `side_number` (like WML without the `$`)

Reply:
I'd say `turn_number` and `side_number`.